docs/reference/elasticsearch/rest-apis/retrievers/knn-retriever.md

```diff
@@ -41,6 +41,15 @@ A kNN retriever returns top documents from a [k-nearest neighbor search (kNN)](d
     The number of nearest neighbor candidates to consider per shard. Needs to be greater than `k`, or `size` if `k` is omitted, and cannot exceed 10,000. {{es}} collects `num_candidates` results from each shard, then merges them to find the top `k` results. Increasing `num_candidates` tends to improve the accuracy of the final `k` results. Defaults to `Math.min(1.5 * k, 10_000)`.
 
 
+```{applies_to}
+stack: ga 9.2
+```
+`visit_percentage`
```

Comment:
```suggestion
`visit_percentage` {applies_to}`stack: ga 9.2`
```

Just fix to use inline tag instead of [section-level tag](https://elastic.github.io/docs-builder/syntax/applies/#section-level) which should only be used after a sub-heading :)